jck/compiler.api/playlist.xml

```diff
@@ -81,6 +81,11 @@
 	</test>
 	<test>
 		<testCaseName>jck-compiler-api-signaturetest</testCaseName>
+		<disabled>
+			<comment>Temporarily disabled on aix on JDK8 for test setup issue: backlog/issues/506</comment>
+			<plat>ppc64_aix(_mixed)?</plat>
```

Comment:
am presuming since you report internal Grinder passes that the value in the plat tag doesn't cause TKG to choke